tests/plugins/clipboard/manual/pasteimagedisable.html

```diff
@@ -0,0 +1,32 @@
+<div id="editor">
+	<p>Paste image here</p>
+	<p></p>
+</div>
+
+<br><br>
+
+<div id="log">Log output:</div>
+
+<script>
+	( function() {
+		if ( !window.FileReader ) {
```

Comment:
This test should be ignored in Internet Explorer 10+ as these browsers support File API, but do not support custom MIME types during pasting (only during drag and drop). Probably the most sensible way to ignore this test there would be to do some browser sniffing. Additionally, this test should be ignored on mobile as pasting images is not possible in the same way as on desktops.

So the check should look like this:

```suggestion
		if ( CKEDITOR.env.ie || bender.tools.env.mobile ) {
```

Could you update it?